viz_scripts/plots.py

```diff
@@ -58,36 +58,15 @@ def format_pct(pct, values):
 
 
 def pie_chart_mode(plot_title,labels,values,file_name):
-    all_labels= ['Gas Car, drove alone',
-                 'Bus', 
-                 'Train', 
-                 'Free Shuttle',
-                 'Taxi/Uber/Lyft', 
-                 'Gas Car, with others', 
-                 'Bikeshare',
-                 'Scooter share',
-                 'E-bike', 
-                 'Walk', 
-                 'Skate board', 
-                 'Regular Bike', 
-                 'Not a Trip',
-                 'No Travel', 
-                 'Same Mode', 
-                 'E-car, drove alone',
-                 'E-car, with others',
-                 'Air',
-                 'Other']
-
-    val2labeldf = pd.DataFrame({"labels": labels, "values": values})
     
-    colours = dict(zip(all_labels, plt.cm.tab20.colors[:len(all_labels)]))
+    colors = dict(zip(labels, plt.cm.tab20.colors[:len(labels)]))
     fig, ax = plt.subplots(figsize=(10, 10), subplot_kw=dict(aspect="equal"))
 
     m_labels, m_values = merge_small_entries(labels, values)
     
     wedges, texts, autotexts = ax.pie(m_values,
                                       labels = m_labels,
-                                      colors=[colours[key] for key in labels],
+                                      colors=[colors[key] for key in labels],
```

Comment:
It would be good to avoid unnecessary changes unless we are explicitly refactoring code.
Please revert unless this is actually necessary.